openmp-17.yaml

```diff
@@ -0,0 +1,71 @@
+package:
+  name: openmp-17
+  version: 17.0.5
+  epoch: 0
+  description: "LLVM OpenMP library"
+  copyright:
+    - license: Apache-2.0
+
+environment:
+  contents:
+    packages:
+      - wolfi-base
+      - busybox
+      - ca-certificates-bundle
+      - build-base
+      - cmake
+      - samurai
+      - binutils-dev
+      - libffi-dev
+      - zlib-dev
+      - libxml2-dev
+      - pkgconf
+      - python3
+      - help2man
+      - llvm17
+      - llvm-cmake-17
+      - llvm17-dev
+      - libLLVM-17
+
+pipeline:
+  - uses: fetch
```

Comment:
suggestion: we could use git-checkout here, which sounds like the ideal choice when possible.

Reply:
will address this in the upcoming pr